modules/addTask.js

```diff
@@ -0,0 +1,14 @@
+import { getTaskFromLocalStorage, setTaskIntoLocalStorage } from './localStorage.js';
+
+const addTask = (text) => {
+  let tasks = getTaskFromLocalStorage();
+  tasks = tasks.sort((v1, v2) => v1.index - v2.index);
+  let index;
+  if (tasks.length === 0) index = 1;
+  else index = tasks[tasks.length - 1].index + 1;
+  const obj = { index, description: text, completed: false };
+  tasks.push(obj);
+  setTaskIntoLocalStorage(tasks);
+};
```

Comment:
- [ ] Please make sure that deleting a task updates all remaining items' indexes, so they represent the current list order and are unique. For example you had task1 with index 1, task2  with index 2, after you delete task1, task2 should get index 1
so you will always have index starting from 1...

https://user-images.githubusercontent.com/77038610/160114243-4624b9b6-b35c-4faa-ad67-8be3db1921f2.mp4